### DISCUSSION TITLE
[Feature] Ability to save and use multiple http ip addresses

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9011] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, My laptop gets one of the 2 ip addresses assigned. Please provide the ability to set and use one of the multiple ip addresses. Thx.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #308 / #2335
